terraform/modules/network/resources.tf

```diff
@@ -26,56 +26,63 @@ resource "aws_security_group" "default" {
       from_port   = 0
       to_port     = 0
       protocol    = "-1"
-      cidr_blocks = concat(split(",", var.config.ip_whitelist), ["10.0.0.0/16"])
+      cidr_blocks = ["10.0.0.0/16"]
+    }
+
+   ingress {
+      from_port   = -1
+      to_port     = -1
+      protocol    = "icmp"
+      cidr_blocks = split(",", var.config.ip_whitelist)
     }
 
    ingress {
       from_port   = 22
       to_port     = 22
       protocol    = "tcp"
-      cidr_blocks = concat(split(",", var.config.ip_whitelist), ["10.0.0.0/16"])
+      cidr_blocks = split(",", var.config.ip_whitelist)
     }
 
    ingress {
       from_port   = 8000
       to_port     = 8000
       protocol    = "tcp"
-      cidr_blocks = concat(split(",", var.config.ip_whitelist), ["10.0.0.0/16"])
+      cidr_blocks = split(",", var.config.ip_whitelist)
```

Comment:
Just wondering if you tested the code with the plain `split`. This attribute expects a list, so at least I would enclosure this between brackets.

I'm fine with these changes, though I remember Patrick having an issue letting 10.0.0.0/16 to access everything. I personally don't mind, but at the same time I don't understand which other machines are in the same network.

Reply:
Yeah, I tested it. `split` returns a list.

> I remember Patrick having an issue letting 10.0.0.0/16 to access everything

I don't see any issue with this but I'm open to alternatives.

> I don't understand which other machines are in the same network

The VPC is created by terraform and only contains the instances built for your attack range.

Reply:
Then I'm all down ;)